src/lib/tabs/tab-group.spec.ts

```diff
@@ -186,6 +186,25 @@ describe('MdTabGroup', () => {
       expect(tabs[1].isActive).toBe(false);
       expect(tabs[2].isActive).toBe(true);
     });
+
+    it('should set the disabled flag on tab', () => {
+      fixture.detectChanges();
+
+      const tabs = fixture.componentInstance.tabs.toArray();
+
+      tabs[0].disabled = false;
+      fixture.detectChanges();
+
+      let labels = fixture.debugElement.queryAll(By.css('.mat-tab-disabled'));
+      expect(labels.length).toBe(0);
+
+      tabs[0].disabled = true;
+      fixture.detectChanges();
+
+      expect(tabs[0].disabled).toBe(true);
+      labels = fixture.debugElement.queryAll(By.css('.mat-tab-disabled'));
+      expect(labels.length).toBe(0);
```

Comment:
Hi @amcdnl! Shouldn't this be `.toBe(1)` instead of `toBe(0)`? I'm not the really the best about testing, so take this more as a question that a point..

Reply:
Good catch, I wrote a couple more tests that are better while I was at it.